scripts/fast-reboot

```diff
@@ -125,6 +128,12 @@ function parseOptions()
             u )
                 SSD_FW_UPDATE_BOOT_OPTION=yes
                 ;;
+            n )
```

Comment:
I'll have the default be yes in the internal images, since we want to make sure this feature is present internally. Because of this, I have two options that'll set this value consistently, regardless of whether a public image or internal image is being tested.